src/farmit/__init__.py

```diff
@@ -245,10 +246,26 @@ def update_changelog(args: Namespace, changelog_path: str, entry: str):
     else:
         logger.warning("CHANGELOG.md is already up-to-date")
 
+def get_current_release(repo: Repo):
+    """Get the current release based on last tag."""
+
+    # Check for any tags
+    if not repo.tags:
+        return None
+
+    next_version, _ = get_version().split('.dev')
+    next = Version(next_version)
```

Comment:
Could be a one-liner:

```suggestion
    next = Version(get_version().split('.dev'))  # Version returns next micro release
```